invenio/templates/configurations/uwsgi.yaml

```diff
@@ -6,31 +6,47 @@ metadata:
 data:
   uwsgi.ini: |
     [uwsgi]
-    stats = 0.0.0.0:9000
+    stats = /tmp/stats.socket
     socket = 0.0.0.0:5000
     module = invenio_app.wsgi:application
     master = true
+    vacuum = true
+    enable-threads = true
     processes = {{ .Values.web.uwsgi.processes }}
     threads = {{ .Values.web.uwsgi.threads }}
-    single-interpreter = true
-    need-app = true
-    disable-logging = true
+    thunder-lock = true # https://marc.info/?l=uwsgi&m=140473636200986&w=2
+                        # https://uwsgi-docs.readthedocs.io/en/latest/articles/SerializingAccept.html
+
+    # disable-logging = true
     log-4xx = true
     log-5xx = true
-    auto-procname = true
-    buffer-size =  10240
-    post-buffering = true
+    log-ioerror = true
+    log-x-forwarded-for = true
+
+    single-interpreter = true
+    need-app = true
+    lazy = true
     lazy-apps = true
-    # thunder-lock = true # https://marc.info/?l=uwsgi&m=140473636200986&w=2
-                          # https://uwsgi-docs.readthedocs.io/en/latest/articles/SerializingAccept.html
+
+    # Silence write errors for misbehaving clients
+    # https://github.com/getsentry/raven-python/issues/732
+    # ignore-sigpipe = true
```

Comment:
Just to be sure, these were added due to `IOError` but are commented. Is that ok?

Reply:
This is a copy/paste from what has been set in ZenodoRDM OpenShift.
Any settings should be/will be re-checked.